src/backend/interfaces/REST/go.server/operations/users/user.go

```diff
@@ -57,6 +58,97 @@ func PatchUser(ctx *gin.Context) {
 
 }
 
+// RequestPasswordReset handles an anonymous POST request on /passwords/reset/ with json payload
+// it will try to trigger a password reset procedure
+func RequestPasswordReset(ctx *gin.Context) {
+	var payload PasswordResetRequest
+	err := ctx.BindJSON(&payload)
+	if err != nil {
+		e := swgErr.New(http.StatusUnprocessableEntity, err.Error())
+		http_middleware.ServeError(ctx.Writer, ctx.Request, e)
+		ctx.Abort()
+		return
+	}
+
+	if payload.RecoveryMail == "" && payload.Username == "" {
+		e := swgErr.New(http.StatusUnprocessableEntity, "neither username nor recovery email provided, at least one required")
```

Comment:
it's a Bad request (400) not an unprocessable entity (422) HTTP error code to return